docs/setup-cluster/deploy-cluster/installation-options.rst

```diff
@@ -75,9 +192,9 @@ Examples:
 
 .. _firewall-rules:
 
-********************************
- Step 4 - Set up Firewall Rules
-********************************
+***********************
+ Set Up Firewall Rules
+***********************
```

Comment:
These feel like they belong with certain master installation information or agent installation information, not here in Installation Options.

Reply:
added a page to keep the content for #7763